deploy/jenkins/jobs/Provision/jobs/dev/jobs/DataPipeline/jobs/Kafka/config.xml

```diff
@@ -77,6 +77,12 @@ return &quot;&quot;&quot;&lt;b&gt;This parameter is not used&lt;/b&gt;&quot;&quo
           <choiceType>ET_FORMATTED_HTML</choiceType>
           <omitValueField>true</omitValueField>
         </org.biouno.unochoice.DynamicReferenceParameter>
+        <hudson.model.StringParameterDefinition>
+          <name>remote</name>
+          <description></description>
+          <defaultValue>processing-cluster-kafka</defaultValue>
```

Comment:
what is this default? @Kaali09

Reply:
We have variablised the hostname in the kafka provision playbook.yml. So default is processing-cluster-kafka.